jobs/README.md

```diff
@@ -0,0 +1,30 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
```

Comment:
The readme should only point directly to the cloud.google.com documentation, rather than repeating it here.

(This is new guidance and part of https://github.com/GoogleCloudPlatform/golang-samples/issues/530.)

Reply:
Sorry, I don't fully understand how to change it. Do you have an example?

Reply:
See https://github.com/GoogleCloudPlatform/golang-samples/pull/535/files. The README should say:

```
See https://cloud.google.com/job-discovery/docs/quickstart-jobs-and-companies.
```

(Or whatever site these samples will show up on.)

Reply:
Done